Ethereal Artefacts/Ethereal Artefacts/Views/HomepageView.swift

```diff
@@ -7,14 +7,18 @@
 
 import SwiftUI
 
-struct HomepageView: View {    
+struct HomepageView: View {
+    @State private var searchText = ""
+    
     var body: some View {
-        NavigationView() {
+        NavigationStack() {
```

Comment:
Only the first page in a flow should have a `NavigationStack` - in your case this is the Login Page. All pages you navigate to start off from `VStack`s.

Reply:
Done.